flytekit-java/src/test/java/org/flyte/flytekit/SdkBindingDataTest.java

```diff
@@ -86,15 +84,16 @@ public void testOfStringCollection() {
     input.add(SdkBindingData.ofString("1"));
     input.add(SdkBindingData.ofString("2"));
 
-    List<String> expected = new ArrayList<>();
-    expected.add("1");
-    expected.add("2");
+    List<String> expectedValue = new ArrayList<>();
+    expectedValue.add("1");
+    expectedValue.add("2");
```

Comment:
```suggestion
    List<String> expectedValue = List.of("1", "2");
```
I think that we are in java 11, so we can use this now

Reply:
here and the rests of the tests